src/db.ts

```diff
@@ -56,43 +57,41 @@ class DbHandler {
         const roomModel = getModelForClass(RoomSchema);  
         try {
             const res = await roomModel.updateOne({ _id: roomId }, updateData);
-            console.log(`Room updated with id (handler): ${roomId}`);
+            console.log(`Room successfully updated with id: ${roomId}, collection: ${roomModel.collection.collectionName}, updated fields: ${Object.keys(updateData).join(', ')}`);
         } catch (error) {
-            console.error("Room update error (handler):", error);
+            console.error(`Failed to update room fields: ${Object.keys(updateData).join(', ')}, with id: ${roomId}, collection: ${roomModel.collection.collectionName}, error:`, error);
         }
     }
 
-    public async createQuiz(quizData: QuizSchema): Promise<void> {
+    public async createQuizQuestionIds(quizData: QuizSchema): Promise<void> {
```

Comment:
[NIT] why did you change the name?

Reply:
wasn't sure if it was better for clarity to keep it short vs keep it consistent between the quiz functions. I assume the former?

Reply:
its your choice at the end of the day, but just make sure this question is answered clearly: if someone who is seeing this code for the first time, will they be able to understand what this function does only by reading the function name?

to me `createQuiz` achieves this, whereas `createQuizQuestionIds` confuses me.